back/lib/tasks/single_use/20240613_rename_gv_profile.rake

```diff
@@ -0,0 +1,30 @@
+namespace :gv_transition do
+  desc 'Replace the Citizenlab default moderator with Go Vocal attributes.'
+  task :rename_profile, [] => [:environment] do
+    reporter = ScriptReporter.new
+    Tenant.safe_switch_each do |tenant|
+      default_moderators = User.where(email: %w[moderator@citizenlab.co moderator@govocal.com])
+      default_moderators.each do |moderator|
+        if moderator.first_name.downcase == 'citizenlab'
+          old_v = moderator.first_name
+          new_v = 'Go Vocal'
+          moderator.first_name = new_v
+          # moderator.remote_avatar_url = 'https://res.cloudinary.com/citizenlab-assets/image/upload/v1623680000/govocal-logo.png'
```

Comment:
Still to be determined (https://go-vocal.slack.com/archives/C06CJ4D3B0R/p1718297440045329)